### PR TITLE
Feature/form

### PR DESCRIPTION
**Pull request by aliceabarca**
What I did:
- created a form for users to fill out and submit.
- added Form component to App.

Testing Notes/Extras: 

Did this break anything?
- [x]  No
- [ ]  Yes

Type of change
- [x]  New feature (non-breaking change which adds functionality)
- [ ]  Bug fix (non-breaking change which fixes an issue)
- [ ]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ]  Super small fix (Corrected a typo, removed a comment, etc.)
- [ ]  Added testing for: 
- [ ]  Skip all the other stuff and briefly explain the fix.

Checklist:
- [ ]  If this code needs to be tested, all tests are passing
- [x]  The code runs locally
- [ ]  The code is DRY. If it's not, I tried my best
- [x]  I reviewed my code before pushing

What's next?
- error handling for form.

### CODE REVIEW

**Review by paulina-isabel — APPROVED** [review]
Looking good!

File: src/components/Forms/Form.js
Comment (apete12) [inline src/components/Forms/Form.js:1]: Everything looks great, nice work Alice!!

**Review by apete12 — APPROVED** [review]
Awesome job! Everything looks great to me

**Comment by judy0ye** [comment]
Looks great!

**Pull request merged**